src/tests/components.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { MemoryRouter, BrowserRouter } from 'react-router-dom';
+import store from '../redux/configureStore';
+import Naving from '../Components/Navbar';
+import Home from '../Components/Home';
+import Region from '../Components/Regions/Region';
+import CountryPage from '../Components/Country/Country-page';
+
+describe('Series of tests rendering of each major components', () => {
+  it('Rendering of Navigation bar(Header) correctly', () => {
+    const tree = renderer
+      .create(
+        <MemoryRouter to="/" className="navbar-brand">
+          <Naving />
+        </MemoryRouter>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('Rendering Country page correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <BrowserRouter>
+            <CountryPage />
+          </BrowserRouter>
+        </Provider>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
```

Comment:
- [ ] It would be better to ensure that all our test cases pass. Currently, we have 3 snapshots test failing. Also, remember to test the pure function in our redux directory
![Screenshot from 2022-02-05 05-00-01](https://user-images.githubusercontent.com/46388113/152629373-f9a87790-ad59-4a33-9812-ab68a956a91f.png)